client/src/data_layer_configs/lead_service_lines_by_water_systems_config.ts

```diff
@@ -114,22 +115,23 @@ const popupInfo: PopupInfo = {
       dataType: FeaturePropertyDataType.String,
       optional: true,
     },
-    {
-      label: 'Number of lead connections',
-      name: 'lead_connections_count',
-      dataType: FeaturePropertyDataType.Number,
-    },
-    {
-      label: 'Number of service lines',
-      name: 'service_connections_count',
-      dataType: FeaturePropertyDataType.Number,
-    },
     {
       label: 'Population served',
       name: 'population_served',
       dataType: FeaturePropertyDataType.Number,
     },
   ],
+  computedProperties: [
+    {
+      label: 'Percent lead lines',
+      name: 'percent_lead',
+      dataType: FeaturePropertyDataType.Percentage,
+      calculate: (features: GeoJsonProperties) =>
+        features == null
+          ? null
```

Comment:
should we check that this is not 0? would this give an error or just be 0

Reply:
Good note thanks